src/tests/ftest/datamover/container_copy.py

```diff
@@ -0,0 +1,169 @@
+            Run mdtest again, but this time on cont2 and read the files back.
+        :avocado: tags=all,datamover,hw,large,full_regression
+        :avocado: tags=containercopy,daoscont1_to_daoscont2
+        """
+
+        # test params
+        mdtest_flags = self.params.get("mdtest_flags", "/run/mdtest/*")
+        file_size = self.params.get("bytes", "/run/mdtest/*")
+        # create pool and cont
+        self.add_pool(connect=False)
+        self.create_cont()
+
+        # set and update mdtest command params
+        self.mdtest_cmd.set_daos_params(self.server_group, self.pool,
+                                        self.container[0].uuid)
+        self.mdtest_cmd.flags.update(mdtest_flags[0])
+        self.mdtest_cmd.write_bytes.update(file_size)
+        # run mdtest
+        self.run_mdtest(self.get_mdtest_job_manager_command(self.manager),
+                        self.processes)
+        # create second container
+        self.create_cont()
+
+        # copy data from cont1 to cont2
+        self.run_dcp(self.pool, self.pool, self.container[0], self.container[1])
+
+        # update and run mdtest read on cont2
+        self.mdtest_cmd.set_daos_params(self.server_group, self.pool,
+                                        self.container[1].uuid)
+        self.mdtest_cmd.flags.update(mdtest_flags[1])
+        self.mdtest_cmd.read_bytes.update(file_size)
+
+        self.run_mdtest(self.get_mdtest_job_manager_command(self.manager),
+                        self.processes)
+
+    def test_daoscont_to_posixfs(self):
+        """Jira ID: DAOS-4782.
+        Test Description:
+            Copy data from daos container to external Posix file system
+        Use Cases:
+            Create a pool
+            Create POSIX type container.
+            Run ior -a DFS on cont1.
+            Create cont2
+            Copy data from cont1 to cont2 using dcp.
+            Copy data from cont2 to external Posix File system.
+            (Assuming dfuse mount as external POSIX FS)
+            Run ior -a DFS with read verify on copied directory to verify
+            data.
+        :avocado: tags=all,datamover,hw,large,full_regression
+        :avocado: tags=containercopy,daoscont_to_posixfs
+        """
+
+        # create pool and cont
+        self.add_pool(connect=False)
+        self.create_cont()
+
+        # update and run ior on cont1
+        self.ior_cmd.set_daos_params(self.server_group, self.pool,
+                                     self.container[0].uuid)
+        self.run_ior(self.get_ior_job_manager_command(), self.processes)
+
+        # create cont2
+        self.create_cont()
+
+        # copy from daos cont1 to cont2
+        self.run_dcp(self.pool, self.pool, self.container[0], self.container[1])
+
+        # start dfuse
+        self.start_dfuse(self.hostlist_clients, self.pool, self.container)
+
+        # copy from daos cont to posix file system
+        self.run_dcp(self.pool, None, self.container[1], None, True)
```

Comment:
I just want to note that this copies from container0 to container1, and then from container1 back to (dfuse) container0. So the second copy is writing to the same file/data. Since the end result is fine, this is probably okay, but I wonder if in the future we might want separate test cases for copying to a _**new**_ dfuse file vs copying to an _**existing**_ dfuse file.
But, until DAOS-5703 is landed, dfuse will fail for new files due to a missing mknod implementation.
I think we can revisit this in the future.